index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>My Portfolio</title>
+  <link rel="stylesheet" href="main.css">
+</head>
+<body>
+  <header>
+    <a href="#"><img src="./pics/LogoPlaceholder.png" alt="logo"></a>
+    <nav>
+      <a href="#"><img src="./pics/Icon-Menu.png" alt="menu"></a>
+    </nav>
+  </header>
+  <main>
+    <section>
+      <h1>Hey there. I'm Edozie. <br> <span id="title-span">I'm a software developer</span></h1>
+      <p id="text">
+        I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.
+      </p>
+      <ul class="socials">
+        <li><a href="#"><img src="/pics/github.png" alt="github logo"></a></li>
+        <li><a href="#"><img src="/pics/linkedin.png" alt="linkedin logo"></a></li>
+        <li><a href="#"><img src="/pics/Enabled.png" alt="enabled"></a></li>
+        <li><a href="#"><img src="/pics/twitter.png" alt="twitter"></a></li>
+        <li><a href="#"><img src="/pics/medium.png" alt="medium"></a></li>
+      </ul>
```

Comment:
- [Optional] The links to your images are broken as the images don't display. please fix this. I think adding a period (.) before the links will fix this issue. Try `./pics/......png`.

Reply:
Thank you @Kamba56 🙏🏾
Will make these changes ASAP